bundle/deploy/terraform/util.go

```diff
@@ -34,6 +34,7 @@ type stateResourceInstance struct {
 
 type stateInstanceAttributes struct {
 	ID   string `json:"id"`
+	Name string `json:"name,omitempty"` // Some resources such as Apps do not have an ID, so we use the name instead.
```

Comment:
Why do we need this?

Reply:
It's for cases when app is removed from bundle config but exists in remote still so we can still correctly display it's name

Reply:
Updated the comment